setup.py

```diff
@@ -27,8 +27,11 @@
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.4',
```

Comment:
I'm glad that you did not forget about trove classifiers. We could also add some for specific django versions since the [official list ](https://pypi.python.org/pypi?%3Aaction=list_classifiers) includes such classifiers.